subsys/mgmt/ec_host_cmd/ec_host_cmd_handler.c

```diff
@@ -161,6 +185,35 @@ int ec_host_cmd_send_response(enum ec_host_cmd_status status,
 	struct ec_host_cmd *hc = &ec_host_cmd;
 	struct ec_host_cmd_tx_buf *tx = &hc->tx;
 
+#ifdef CONFIG_EC_HOST_CMD_IN_PROGRESS_STATUS
+	if (cmd_in_progress) {
+		/* We previously got EC_HOST_CMD_IN_PROGRESS. This must be the completion
+		 * of that command, so save the result code.
+		 */
+		LOG_INF("HC pending done, size=%d, result=%d",
+			args->output_buf_size, status);
+
+		/* Don't support saving response data, so mark the response as unavailable
+		 * in that case.
+		 */
+		if (args->output_buf_size != 0)
+			saved_status = EC_HOST_CMD_UNAVAILABLE;
+		else
+			saved_status = status;
```

Comment:
add braces (surprised the CI did not catch it...)

Reply:
Done.